CMakeLists.txt

```diff
@@ -19,6 +19,19 @@ option(USE_AFLPLUSPLUS "Use AFL++ instead of libFuzzer" OFF)
 option(ENABLE_DEBUG "Enable debug build with full symbols" OFF)
 option(FORCE_STATIC_LINKING "Force static linking of all dependencies" OFF)
 
+option(USE_VENDORED_CPPSQLITE "Use the bundled version of cpp-sqlite" ON)
+option(USE_VENDORED_CPPZMQ "Use the bundled version of cppzmq" ON)
+option(USE_VENDORED_FLATBUFFERS "Use the bundled version of flatbuffers" ON)
+option(USE_VENDORED_LEXY "Use the bundled version of lexy" ON)
+option(USE_VENDORED_MINICORO "Use the bundled version of minicoro" ON)
+option(USE_VENDORED_MINITRACE "Use the bundled version of minitrace" ON)
+option(USE_VENDORED_TINYXML2 "Use the bundled version of tinyxml2" ON)
+option(USE_VENDORED_WILDCARDS "Use the bundled version of wildcards" ON)
+
+set(BTCPP_LIB_DESTINATION     lib)
```

Comment:
This code was repeated on both `conan_build.cmake` and `ament_build.cmake`. I've unified it here.